setup.py

```diff
@@ -63,10 +64,10 @@
     include_package_data=True,
     install_requires=[
         "cached-property>=1.5.2,<2;python_version<'3.8'",
-        "eth-hash>=0.3.1,<0.6.0",
-        "eth-typing>=3.0.0,<4.0.0",
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.10.1,<1.0.0;implementation_name=='cpython'",
+        "eth-hash>=0.3.1",
+        "eth-typing>=3.0.0",
+        "toolz>0.8.2;implementation_name=='pypy'",
+        "cytoolz>=0.10.1;implementation_name=='cpython'",
     ],
     python_requires=">=3.6,<4",
```

Comment:
We probably want to update this to `>=3.7` although technically I think the python `cimg` for 3.7 uses `3.7.15` 🤔